2_homework/Task_16.py

```diff
@@ -0,0 +1,26 @@
+# Задача 16: Требуется вычислить, сколько раз встречается некоторое число X в массиве A[1..N]. 
+# Пользователь в первой строке вводит натуральное число N – количество элементов в массиве. В последующих  строках записаны N целых чисел Ai. 
+# Последняя строка содержит число X
+
+# *Пример:*
+# 5
+#     1 2 3 4 5
+#     3
+#     -> 1
+
+print('Введите количество элементов в массиве:')
+n = int(input())
+print('Введите число, которое необходимо проверить в массиве:')
+x = int(input())
+count = 0
+
+import random
+list_1 = [random.randint(1,10) for _ in range (n)]
+
+for i in list_1:
+    print(i)
+
+for i in list_1:
+    if i == x:
+        count += 1
```

Comment:
Но это можно сделать и в одном цикле.
```suggestion
for i in list_1:
    print(i, end=' ')
    if i == x:
        count += 1
```